core/src/main/java/com/digitalpebble/stormcrawler/protocol/okhttp/HttpProtocol.java

```diff
@@ -230,6 +231,19 @@ public EventListener create(Call call) {
         // enable support for Brotli compression (Content-Encoding)
         builder.addInterceptor(BrotliInterceptor.INSTANCE);
 
+        Map<String, Object> connectionPoolConf =
+                (Map<String, Object>) conf.get("http.protocol.connection.pool");
```

Comment:
change the config key to 'okhttp.protocol.connection.pool' to make it obvious that it applies only to it?